docs/source/data-access-api/calibration-files.rst

```diff
@@ -0,0 +1,146 @@
+.. _calibration_files:
+
+Calibration Files
+=================
+
+Calibration files are files used in processing that are provided by instrument teams
+to the SDC. These are files which contain values that could change over the course of
+the mission. Although they are called calibration files here, they may not contain calibration data,
+but instead any kind of data which is used in processing, but may infrequently change in time.
```

Comment:
If we are defining the files this way, I wonder if it is better to call them "auxiliary files"?

Just putting it out there for discussion (I think I would still actually prefer to call them calibration files myself, mostly because spelling out "auxiliary" is a nightmare).

Reply:
I agree that "calibration" is a little too specific for the files, but I think I've only ever heard them described as calibration files, regardless of if they are used for calibration. So I stuck with that use case.

Reply:
"ancillary" is another term I've heard used before. But you've explicitly explained it here so this seems like it might be fine to be left as "calibration".